Process/NewServices.md

```diff
@@ -0,0 +1,15 @@
+To enable the ops department to support your new service, we need to know a few things
+
+ 1. If ops needs to escalate issues with this service, who do we contact?
+ 2. How can ops file tickets for this service?  Which JIRA project?
+ 3. What platform is the service built on?  (Language? Framework?)
+ 4. If the service has its own data store, how is it backed up? How does one restore from backup?
+ 5. What is the disaster recovery plan for this service?  Do we need to mirror the infrastructure in our DR site (reston)?
```

Comment:
I think we need a real test that DR plan is actually working and we've got expected result in the second datacenter.